pkg_security.es.Rmd

```diff
@@ -1,81 +1,83 @@
 
-If you use [`vcr`](https://docs.ropensci.org/vcr/) or [`httptest`](https://enpiar.com/r/httptest/) in tests for caching API responses, you need to make sure the recorded requests / fixtures do not contain secrets. Refer to [`vcr` security guidance](https://books.ropensci.org/http-testing/security-chapter.html) and [`httptest` guidance "Redacting and Modifying Recorded Requests"](https://enpiar.com/r/httptest/articles/redacting.html), and inspect your recorded requests / fixtures before committing them the first time to be sure you got the setup right.
+`vcr` Al ser un paquete rOpenSci, puedes publicar cualquier pregunta que tengas en [foro rOpenSci](https://discuss.ropensci.org/).
 
-`vcr` being an rOpenSci package, you can post any question you might have to [rOpenSci forum](https://discuss.ropensci.org/).
+#### Compartir secretos con los servicios de CI
 
-#### Share secrets with CI services
+Ahora, puede que necesites compartir secretos con [servicios de integración continua](#ci).
 
-Now, you might need to share secrets with [continuous integration services](#ci).
+Puedes almacenar las claves de la API como variables de entorno / secretos, consultando la documentación del servicio de IC.
 
-You could store API keys as environment variables / secrets, referring to the docs of the CI service.
+Para más detalles y consejos sobre el flujo de trabajo, consulta [al artículo de Gárgolas "Gestionar los tokens de forma segura"](https://gargle.r-lib.org/articles/articles/managing-tokens-securely.html) y el capítulo [capítulo de seguridad del libro HTTP testing in R](https://books.ropensci.org/http-testing/security-chapter.html).
 
-For more details and workflow advice, refer [to the gargle article "Managing tokens securely"](https://gargle.r-lib.org/articles/articles/managing-tokens-securely.html) and the [security chapter of the HTTP testing in R book](https://books.ropensci.org/http-testing/security-chapter.html).
+Documenta los pasos que has dado en [CONTRIBUCIÓN.md](#friendlyfiles) para que tú, o digamos un nuevo mantenedor, pueda recordar cómo hacerlo la próxima vez.
 
-Document the steps you made in [CONTRIBUTING.md](#friendlyfiles) so you, or say a new maintainer, can remember how to do that next time.
+#### Secretos y colaboraciones
 
-#### Secrets and collaborations
+¿Qué pasa con los pull requests de colaboradores externos? Las pruebas que utilicen tus secretos fallarán a menos que utilices algún tipo de respuesta simulada/en caché, por lo que es posible que quieras omitirlas dependiendo del contexto. Por ejemplo, en tu cuenta CI podrías crear una variable de entorno llamada `THIS_IS_ME` y luego omitir las pruebas en función de la presencia de esta variable. Obviamente, esto significa que las comprobaciones del PR por parte del CI no son exhaustivas, por lo que tendrás que comprobar el PR localmente para ejecutar todas las pruebas.
```

Comment:
I'm confused about this part. If your CI system stored the credentials, then PR tests wouldn't fail, would they? Tests run locally by contributors would.

Reply:
@eliocamp on GitHub Actions (and presumedly other CI services) the credentials are available for PRs started from the same repo but not for PRs from forks. The idea is that if you made a fork of a popular repo and had access to the secrets, then you could amend the code to do something bad (like, updating someone's website, or worse).

In this repository for instance we skip the Netlify preview for PRs from forks because for instance your fork does not have access to the Netlify side ID and token that are stored as GitHub Actions secrets for this repository.

I opened #545 as it's useful feedback so thank you!